File: Java/AutomationFramework/src/com/app/helpers/actions/KeyboardActions.java
package com.app.helpers.actions;

import com.app.helpers.logging.ILogger;
import com.app.helpers.logging.InfoLogger;
import com.aventstack.extentreports.model.Test;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class KeyboardActions {
    WebDriver driver;

    KeyboardActions(WebDriver driver) {
        this.driver = driver;
    }

    public void SendText(WebElement element, String text) {
        element.sendKeys(text);
        ILogger.PrintLog(new InfoLogger().LogMessage("Entered the '"+text+ "' to '" + element + "'"));
    }

    public void ClearText(WebElement element) {
        element.clear();
        ILogger.PrintLog(new InfoLogger().LogMessage("Cleared the value from '" + element + "'"));
    }
}

File: DotNet/Framework/AutomationFramework/Helpers/Actions/KeyBoardActions.cs
using OpenQA.Selenium;
using Helpers.Logging;

namespace Helpers.Actions
{
    public class KeyBoardActions
    {
        IWebDriver driver;

        public KeyBoardActions(IWebDriver driver)
        {
            this.driver = driver;
        }

        public void SendText(By selector, string text)
        {
            ClearText(selector);
            driver.FindElement(selector).SendKeys(text);
            Logger.PrintLog(new InfoLogger().LogMessage($"Entered the '{text}' to the element '{selector}'"));
        }

        public void ClearText(By selector)
        {
            driver.FindElement(selector).Clear();
            Logger.PrintLog(new InfoLogger().LogMessage($"Cleared the values inside the element '{selector}'"));
        }
    }
}
